public/app/core/components/TopNav/TopSearchBar.tsx

```diff
@@ -0,0 +1,90 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { FilterInput, Icon, Tooltip, useStyles2 } from '@grafana/ui';
+import { contextSrv } from 'app/core/core';
+
+import { TOP_BAR_LEVEL_HEIGHT } from './types';
+
+export function TopSearchBar() {
```

Comment:
not sure what the best name for this "top level" is. GlobalBar?  TopTopBar :rofl: